### PR TITLE
EM-1316: Small Biz Compare Chart

### PR DESCRIPTION
**Pull request by arelia**
Several updates to tables
* Update table styles per [EM-1316](https://cb-content-enablement.atlassian.net/browse/EM-1316) from spaced columns to borders

**Before:**
image

**After:**
image

* Update `table--product` and `table--ecom` mixins to mostly just handle colors
* New `table__column-highlight` mixin for column highlight
* Rearranged style sheet so main components are on top, modifiers on bottom
* New variables for tables, font size and text color

Jira: https://cb-content-enablement.atlassian.net/browse/EM-1316
Employer PR: https://github.com/cbdr/employer/pull/872
QA: http://web.employer-7.development.c66.me/recruiting-solutions/small-business-subscription-plans-and-pricing

### CODE REVIEW

File: sass/directives/_tables.scss [hunk 1]
```diff
@@ -38,76 +38,146 @@
...
+    color: white;
+    text-align: center;
+    position: relative;
+    background-clip: padding-box;
+  }
 
-    th:nth-of-type(3n) {
-      background-color: $table-column-heading-3;
+  th[scope='row'] {
+    padding: $table--row-column-headers--row-top-bottom-padding $base-spacing-medium;
```
Comment (mrfantasticwonders) [inline sass/directives/_tables.scss]: `$table--row-column-headers--row-top-bottom-padding` 0.o

Maybe `$column-headers--top-bottom-padding`? Just a really long variable name.

**Comment by mrfantasticwonders** [comment]
Checks out in Safari (latest), Firefox (latest), Chrome (latest, and IE (11,13,14)

File: sass/directives/_tables.scss (hunk 1 above)
Comment (arelia) [inline sass/directives/_tables.scss]: yeah it is a long variable name :( but the name reflects a table > with the row-and-column-headers style (as opposed to the other type of table on the [ecom page](https://hiring.careerbuilder.com/promotions) which just has column headers:

image

that table above uses mixins that are generically named `table` and those need to be refactored to use a modifier as well, so it's clear what each set of mixins is used for

and `row-top-bottom-padding` reflects that this padding isn't what's used on the top and bottom of the column headers, it's only used on the top and bottom of rows, including the `th`s and the `td`s within those rows

Comment (arelia) [inline sass/directives/_tables.scss]: I could see using `table--row-column-headers` as the name for that type of mixin and `row-column-headers-table--row-top-bottom-padding` or `row-column-headers-table__row-top-bottom-padding` as the name for this variable to remove one level of naming

**Pull request merged**